File: frontend-pantry-android/app/src/main/java/com/zancheema/android/pantry/common/ViewUtils.java
package com.zancheema.android.pantry.common;

import android.view.View;

import com.google.android.material.snackbar.BaseTransientBottomBar;
import com.google.android.material.snackbar.Snackbar;

public class ViewUtils {
    public static void showSnackbar(View view, String message) {
        Snackbar
                .make(view, message, BaseTransientBottomBar.LENGTH_LONG)
                .show();
    }
}

File: frontend-pantry-android/app/src/main/java/com/zancheema/android/pantry/common/ViewUtils.kt
package com.zancheema.android.pantry.common

import android.view.View
import com.google.android.material.snackbar.BaseTransientBottomBar
import com.google.android.material.snackbar.Snackbar

fun View.showSnackbar(message: String) {
    Snackbar
        .make(this, message, BaseTransientBottomBar.LENGTH_LONG)
        .show()
}